### PR TITLE
Understanding Database Architecture and Models

### PR DESCRIPTION
**Pull request by kvardhan10**
Learn the concepts of database and its three types of built architecture. Study the different aspects of the Hierarchical Data model, Network Data model and Relational Data model.

This commit includes one index.md file and one hero.jpg file.
(Requesting the team to review the article on 'Understanding Database Architecture and Models').

### CODE REVIEW

**Comment by hectorkambow** [comment]
👋 Good afternoon @sudosoph 
Wanted to ask if you could give this "Understanding Database Architecture and Models" article a Tech review when you have a moment - I've given it a preliminary review - added the front matter - and plagiarism check is done. 

Feel free to add any comments that you think would help the author add some more value. 
🚀

**Comment by hectorkambow** [comment]
Plagiarism check done

File: articles/understanding-dbms/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+images:
+
+  - url: /engineering-education/understanding-dbms/hero.jpg
+    alt: Database Architecture and Models example image
+
+---
+Computers have made our lives easier in handling complicated calculations and solving complex problems. Hands down computers are one of the best inventions of mankind. While using computers and instilling them in our lives has made our work a piece of cake, maintaining them can be a difficult to accomplish.
+<!--more-->
+### Central Processing Unit
+Computers are nothing more but a series of electrical and electronic circuits that run together to form a single working unit called the **CPU (short for Central Processing Unit)**. One such component of the CPU is the **Memory Unit**.
```
Comment (Manley12) [inline articles/understanding-dbms/index.md:21]: Please note that computers are more than electrical and electronic circuits that form a single unit that is called the CPU. A computer is a collection of parts, of which one of them is the CPU. You also have the motherboard, GPU, and power supply unit.

File: articles/understanding-dbms/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+date: 2020-09-21T00:00:00-05:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/understanding-dbms/hero.jpg
+    alt: Database Architecture and Models example image
+
+---
+Computers have made our lives easier in handling complicated calculations and solving complex problems. Hands down computers are one of the best inventions of mankind. While using computers and instilling them in our lives has made our work a piece of cake, maintaining them can be a difficult to accomplish.
```
Comment (Manley12) [inline articles/understanding-dbms/index.md:18]: remove "a" from the last sentence section "maintaining them can be a difficult to accomplish"

**Review by Manley12 — APPROVED** [review]
Wow, amazing article! Reminded me of Data Structures class where we learned about trees, graphs, and tables.

I only found one incorrect statement about computers and how they relate to CPUs.

**Comment by hectorkambow** [comment]
@kvardhan10 just pinging you to make sure you saw the comments from our Peer Reviewer 🚀

File: articles/understanding-dbms/index.md (hunk 1 above)
Comment (kvardhan10) [reply articles/understanding-dbms/index.md:21]: Thank you @Manley12 . Glad to hear that you liked it.
That's a great point. I will make the necessary changes. Thanks for letting me know.

**Comment by kvardhan10** [comment]
Yes @hectorkambow  , I did and I've made the required changes. I was hung up on my college work so I forgot to check my mails. 
Sorry! My bad.

**Comment by hectorkambow** [comment]
> Yes @hectorkambow , I did and I've made the required changes. I was hung up on my college work so I forgot to check my mails.
> Sorry! My bad.

No worries what so ever - 🚀  
We understand - school work comes 1st

**Comment by hectorkambow** [comment]
@ninjaginja Good afternoon Molly 👋 
Wanted to ask if you could give this "Understanding Database Architecture and Models" article a final tech review - It has been reviewed by one of our newest Peer Reviewers and but I'd like to get a double-check the tech accuracy before we publish.
All in an effort to help educate our newer Peer Reviewers on the revisions process and what they should be looking for moving forward.
Thank you 🚀

**Pull request merged**